velocity-tools-examples/velocity-tools-examples-showcase/src/main/webapp/WEB-INF/web.xml

```diff
@@ -19,10 +19,13 @@
  under the License.
 -->
 
-<web-app xmlns="http://xmlns.jcp.org/xml/ns/javaee" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_3_1.xsd"
-         version="3.1">
-  <!-- servlets -->
+<web-app
+        xmlns="http://xmlns.jcp.org/xml/ns/javaee"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_4_0.xsd"
+        version="4.0"
+>
```

Comment:
Note: This is technically not required, they will run just fine on a new container...

Reply:
Yes, added just for completeness